test/spec/utils/fetchers/fetcher_spec.rb

```diff
@@ -0,0 +1,113 @@
+require_relative '../../spec_helper'
+
+class FakeFetcher
+  extend Util::Fetcher
+
+  attr_reader :id, :date
+
+  def initialize(fields={})
+    @id = fields['id']
+    @date = fields['date']
+    @date = Date.parse(date) if date.is_a? String
```

Comment:
I had that in the beginning, but it's always coming out of `fields['date']` so it looked not as pretty I thought:

```ruby
    @date = fields['date'].is_a?(String) ? Date.parse(fields['date']) : fields['date']
```

I'm open to changing it, I just didn't think it looked as nice. Let me know.